### DISCUSSION TITLE
how to reply in private

### DISCUSSION THREAD

**Discussion by ngdream** [python-telegram-bot/python-telegram-bot#3472] [Q&A]
please how to reply in private chat

**Comment by clot27** [comment]
see the parameter `reply_to_message_id` of [telegram.Bot.send_message](https://docs.python-telegram-bot.org/telegram.bot.html#telegram.Bot.send_message).  Also the parameter `quote` of [telegram.Message.reply_text](https://docs.python-telegram-bot.org/telegram.message.html#telegram.Message.reply_text)
